cli/src/main.rs

```diff
@@ -271,7 +271,7 @@ fn run() -> error::Result<()> {
                 debug,
                 debug_graph,
                 debug_xml,
-                Some(&cancellation_flag),
+                Some(cancellation_flag.clone()),
```

Comment:
Do we need the `.clone()` here?

Reply:
Yes, we need it cause there is a loop above on line [258](https://github.com/tree-sitter/tree-sitter/pull/1070/files#diff-37970e7d817d4316a093bb967d74ff7e26d9c6c01d93f5a79d3d195ba9f6155bL258) and so we can't give exclusive ownership for the `cancellation_flag` to the `parse::parse_file_at_path()` method call just for a one loop iteration.